content/ADR-114-p2p-adapter.md

```diff
@@ -300,122 +307,49 @@ type NewPeerRoutingTableEvent = {
-    } else {
-      for (const p of fromPeerConnections) {
-        if (excluding.has(p)) {
-          continue
-        }
-        let relayedRoute = calculateRouteBeetwen(p, toPeer, [p, ..._excluding])
-        if (relayedRoute !== "server") {
-          relayedRoute = [p, ...relayedRoute]
-          if (route === "server" || route.length > relayedRoute.length) {
-            route = relayedRoute
-          }
-        }
-      }
-    }
-
-    calculatedRoutes.set(toPeer, route)
-
-    // NOTE: routes are bidirectional
-    getOrCreateRoutingTable(toPeer).set(fromPeer, route === "server" ? route : Array.from(route).reverse())
-    return route
-  }
-
-  const peers = new Set<string>()
-
-  for (const [peer, connections] of mesh) {
-    peers.add(peer)
-    for (const connection of connections) {
-      peers.add(connection)
-    }
-  }
-
-  for (const peerFrom of peers) {
-    for (const peerTo of peers) {
-      if (peerFrom === peerTo) {
-        continue
-      }
-
-      calculateRouteBeetwen(peerFrom, peerTo, [])
-    }
-  }
-
-  return routingTables
-}
-```
-
 ## Packet
 
 A packet contains a source (peer id) and a target specifying to whom the packet is for and the route to follow to reach it.
 
 ```typescript
+
 type Packet = {
   source: Address
   target: Route[]
 }
 ```
 
-## Peer
+## Peer 
 
 Each time a package needs to be sent, then the peer will create it using its own paths as routing table and will make the network to obey that flow.
 
-```json
-{
-  "source": "peer1",
-  "target": [["peer2", "peer3"]]
+```typescript
+{ 
+  source: 'peer1';
+  target: [ ['peer2', 'peer3'] ];
 }
 ```
 
 This means each peer will check the `target` field, always the message will be processed, and then it will be relayed in the way the route value indicates.
 
-Notice the peer relaying the message is not expected to remove itself either as a recipent or as a hop in the route, since this will require encoding the package again.
+Notice the peer relaying the message is not expected to remove itself either as a recipient or as a hop in the route, since this will require encoding the package again. 
 
 # Benefits
 
 - Since there is a specific routing table for each message, there is no need to expire message or count hops. If a route is cut, the message will not be delivered by the mesh, this means the message should be relayed using the messaging service (if reliable) or discarded (if unreliable).
 - The messaging service fallback provide a safety guarantee against network cluster partitions.
 - A given implementation can be optimized by suggesting peers to connect to certain others in order to avoid clustering and minimize messaging service usage. This is out of the scope for this document.
-- Since the routing service will know the status of the mesh at all times, it's easy to graph and debug network problems.
+- Since the routing service will know the status of the mesh at all times, it's easy to graph and debug network problems. 
```

Comment:
there is a mention to the routing service here

Reply:
done